lerna.json

```diff
@@ -9,9 +9,9 @@
   "command": {
     "version": {
       "yes": true,
-      "push": true,
+      "push": false,
       "message": "chore: publish %s",
-      "allowBranch": "master"
+      "allowBranch": "*"
```

Comment:
Do we really need to change this? `version` will be called in the master branch in order to open the publish PR, won't it?

If we really need to set this, we should limit the release to all branches beginning with e.g. `release-...`. It's a glob pattern so it should be possible.